content/gsoc/2025/_index.md

```diff
@@ -52,26 +46,26 @@ The type system design will require particular attention to several key areas. T
 
 ### Potential mentors
 
-
-* [Saransh Chopra](https://Saransh-cpp.github.io/)
-* [Agriya Khetarpal](https://github.com/agriyakhetarpal/)
+- [Saransh Chopra](https://Saransh-cpp.github.io/)
```

Comment:
I won't be available this summer to mentor projects. Can we please remove me in this PR itself? 😞 
```suggestion
```

Reply:
Thanks for letting me know, @Saransh-cpp! I'll wait until we have a resolution on the Slack thread you posted, and will remove you by Thursday/Friday. I won't be available as a primary mentor either, so we could either drop the project or mark it as "tentative".

I'll merge this PR at the moment without this suggestion so that we can indicate the availability of the open project idea – I hope that's fine!